pom.xml

```diff
@@ -10,7 +10,7 @@
 
   <groupId>com.meltmedia.jgroups</groupId>
   <artifactId>jgroups-aws</artifactId>
-  <version>1.5.1-SNAPSHOT</version>
+  <version>2.0.0-SNAPSHOT</version>
```

Comment:
Ok! Took a guess that the move from 1.7 to 1.8 was to consider breaking, but I can change!